### DISCUSSION TITLE
Bluesky should support inline translations via DeepL

### DISCUSSION THREAD

**Discussion by Zero3K** [bluesky-social/atproto#1376] [Bluesky Client App]
*subject*

Instead of using Google Translate since it is more accurate.

**Comment by Zero3K** [comment]
I like the way the official extension does it on Twitter.

**Comment by Zero3K** [comment]
does anyone have any thoughts about it?

**Comment by ryvwel** [comment]
You should ask this at the client repo (`social-app`), not the server

**Comment by Zero3K** [comment]
Okay. I made it as a feature request in its Issue Tracker.
